lighthouse-core/fraggle-rock/config/filters.js

```diff
@@ -47,17 +49,32 @@ function filterAuditsByAvailableArtifacts(audits, availableArtifacts) {
 function filterCategoriesByAvailableAudits(categories, availableAudits) {
   if (!categories) return categories;
 
-  const availableAuditIds = new Set(availableAudits.map(audit => audit.implementation.meta.id));
+  const availableAuditIdToMeta = new Map(
+    availableAudits.map(audit => [audit.implementation.meta.id, audit.implementation.meta])
+  );
 
-  return Object.fromEntries(
-    Object.entries(categories).map(([categoryId, category]) => {
+  const categoryEntries = Object.entries(categories)
+    .map(([categoryId, category]) => {
       const filteredCategory = {
         ...category,
-        auditRefs: category.auditRefs.filter(ref => availableAuditIds.has(ref.id)),
+        auditRefs: category.auditRefs.filter(ref => availableAuditIdToMeta.has(ref.id)),
       };
+
+      const didFilter = filteredCategory.auditRefs.length < category.auditRefs.length;
+      const hasOnlyManualAudits = filteredCategory.auditRefs.every(ref => {
+        const meta = availableAuditIdToMeta.get(ref.id);
+        if (!meta) return false;
+        return meta.scoreDisplayMode === Audit.SCORING_MODES.MANUAL;
+      });
+
```

Comment:
that line is empty of course it's covered... 

go home codecov you're drunk 😆 

![image](https://user-images.githubusercontent.com/2301202/115245480-935bf200-a0ea-11eb-9ac8-393d259d5e1a.png)


(I'm assuming this is some source mapping issue because line 66 *can't* be covered, it's a make TS happy check that's impossible)